bioblend/galaxy/client.py

```diff
@@ -154,7 +154,7 @@ def _get(self, id=None, deleted=False, contents=None, url=None,
                 bioblend.log.warning(msg)
                 time.sleep(retry_delay)
 
-    def _post(self, payload, id=None, deleted=False, contents=None, url=None,
+    def _post(self, payload=None, id=None, deleted=False, contents=None, url=None,
```

Comment:
For this to work, `make_post_request()` needs to be slightly modified to make its `payload` parameter optional.
Then the tests should pass.

Reply:
I've rebased that into the last commit.